src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -640,7 +634,7 @@ export function getEventContent(event) {
       };
   }
 }
-
+// TODO: The guidance for the at_vso status differs for the rest, should it?
```

Comment:
@gnakm The first paragraph here includes an additional sentence for `at_vso` ("Once your representative has completed their review, your case will be returned to the Board. "), should it include this additional, initial sentence? Or should it be the same as the other usages of `DECISION_REVIEW_CONTENT` (`onDocket`, `decisionInProgress`, `bvaDevelopment`, etc.)

Reply:
Yeah, `at_vso` is an odd use case and will have an alert that follows so it's different than others. Good 👀though!

Reply:
Thanks for the context @gnakm.